### PR TITLE
Allow DateRangePickerInput border theme to be overriden

### PR DESCRIPTION
**Pull request by mmahalwy**

### CODE REVIEW

File: src/components/SingleDatePickerInput.jsx [hunk 1]
```diff
@@ -217,7 +217,10 @@ export default withStyles(({ reactDates: { color } }) => ({
   },
 
   SingleDatePickerInput__withBorder: {
-    border: `1px solid ${color.core.border}`,
+    borderColor: color.border,
+    borderWidth: border.pickerInput.borderWidth,
+    borderStyle: border.pickerInput.borderStyle,
+    borderRadius: border.pickerInput.borderRadius,
```
Comment (mmahalwy) [inline src/components/SingleDatePickerInput.jsx]: Maybe this is not needed? Not sure

File: src/components/DateRangePickerInput.jsx [hunk 2]
```diff
@@ -296,8 +296,10 @@ export default withStyles(({ reactDates: { color, sizing } }) => ({
   },
 
   DateRangePickerInput__withBorder: {
-    border: `1px solid ${color.border}`,
```
Comment (ljharb) [inline src/components/DateRangePickerInput.jsx]: minor; but changing `border` might be a breaking change for those overriding it with CSS

Comment (mmahalwy) [reply src/components/DateRangePickerInput.jsx]: I thought in the most recent version of react-dates the classNames are scrambled, no?

Comment (ljharb) [reply src/components/DateRangePickerInput.jsx]: with the aphrodite interface sure, but i'm not sure about with the css interface

Comment (majapw) [reply src/components/DateRangePickerInput.jsx]: I think anyone overriding the border will see the same effect they did before, so I'm not concerned.

File: src/components/SingleDatePickerInput.jsx (hunk 1 above)
Comment (majapw) [reply src/components/SingleDatePickerInput.jsx]: def is. thank you for doing it in both pickers!

**Comment by mmahalwy** [comment]
awesome, thanks @majapw

**Pull request merged**